jupyterlab/labbox_ephys_widgets_jp/src/extensions/unitstable/unitstable.py

```diff
@@ -0,0 +1,7 @@
+import labbox_ephys as le
```

Comment:
added the UnitsTable to jupyter extension, although the metrics columns are not included at this point